net/tstun/wrap.go

```diff
@@ -947,6 +950,10 @@ func (t *Wrapper) Read(buffs [][]byte, sizes []int, offset int) (int, error) {
 		t.sendBufferConsumed()
 	}
 
+	metricOutboundPacketsTotal.Add(trafficLabel{
+		Action: TrafficActionAccept,
+	}, int64(len(res.data)))
```

Comment:
I'm not certain about this one, I suspect I might need to remove the dropped ones.

Reply:
Yeah, perhaps count dropped packets in a local variable in a loop, and increment both metrics here at the end?